### ISSUE TITLE
Extra blank space in "No. #NUMERIC#" returning unexpected tokens from MosesTokenizer

### ISSUE THREAD

**Issue by alvations** [nltk/nltk#1796]
To resolve the nltk/nltk_data#85, stripping the non-breaking prefixes would emulate the Perl chomp from the original Moses tokenizer.perl

This can be patched easily by replacing https://github.com/nltk/nltk/blob/develop/nltk/tokenize/moses.py#L244

```
# Initialize the language specific nonbreaking prefixes.
 self.NONBREAKING_PREFIXES = nonbreaking_prefixes.words(lang)
```

to 

```
# Initialize the language specific nonbreaking prefixes.
 self.NONBREAKING_PREFIXES = [_nbp.strip() for _nbp in nonbreaking_prefixes.words(lang)]
```

This is good first bug to contribute to NLTK too =)

**Comment by ali-abdullah** [comment]
@alvations 
I've requested a pull request with the changes you suggested. First time contributing to open source, so please let me know if I need to make any changes.

**Comment by alvations** [comment]
Resolved in #1802
